src/Prometheus.Client.HttpClient/Prometheus.Client.HttpClient.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.1;net6.0;net7.0</TargetFrameworks>
+    <SignAssembly>False</SignAssembly>
```

Comment:
Remove this section, let's sign with snk.

Reply:
Signing is already been configured in props. Let me remove this.